packages/aws-rfdk/lib/deadline/lib/render-queue.ts

```diff
@@ -448,6 +449,10 @@ export class RenderQueue extends RenderQueueBase implements IGrantable {
       });
     }
 
+    if (props.securityGroups) {
```

Comment:
I suggest replacing this in favour of:

1. We'll want to modify the `const loadBalancer = new ApplicationLoadBalancer(this, 'LB', {` to pass the frontend security group, from the props, as the `securityGroup` [property](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-elasticloadbalancingv2.ApplicationLoadBalancer.html#securitygroup). All listeners created for the ALB will use the LoadBalancer's security groups.
2. Similarly, we'll want to modify the  `this.asg = this.cluster.addCapacity('RCS Capacity', {` to pass the backend security group. The properties of that call are just AutoScalingGroup properties -- you can see that here: https://github.com/aws/aws-cdk/blob/4553d4b37cc7c2062eebc36bfe7a03b0226b598d/packages/%40aws-cdk/aws-ecs/lib/cluster.ts#L226 .